scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/InProcessTaskExecutor.java

```diff
@@ -130,49 +131,51 @@ public TaskResultImpl execute(TaskContext taskContext, PrintStream output, Print
-                                                                                   schedulerNodeClient,
-                                                                                   IDataSpaceClient.Dataspace.USER);
+                    schedulerNodeClient,
+                    IDataSpaceClient.Dataspace.USER);
             globalSpaceClient = forkedTaskVariablesManager.createDataSpaceNodeClient(taskContext,
-                                                                                     schedulerNodeClient,
-                                                                                     IDataSpaceClient.Dataspace.GLOBAL);
+                    schedulerNodeClient,
+                    IDataSpaceClient.Dataspace.GLOBAL);
 
             forkedTaskVariablesManager.addBindingsToScriptHandler(scriptHandler,
-                                                                  taskContext,
-                                                                  variables,
-                                                                  thirdPartyCredentials,
-                                                                  schedulerNodeClient,
-                                                                  userSpaceClient,
-                                                                  globalSpaceClient,
-                                                                  resultMetadata);
+                    taskContext,
+                    variables,
+                    resultMap,
+                    thirdPartyCredentials,
+                    schedulerNodeClient,
+                    userSpaceClient,
+                    globalSpaceClient,
+                    resultMetadata);
 
             Stopwatch stopwatch = Stopwatch.createUnstarted();
             TaskResultImpl taskResult;
             try {
                 stopwatch.start();
                 Serializable result = execute(taskContext,
-                                              output,
-                                              error,
-                                              scriptHandler,
-                                              thirdPartyCredentials,
-                                              variables);
+                        output,
+                        error,
+                        scriptHandler,
+                        thirdPartyCredentials,
+                        variables);
                 stopwatch.stop();
                 taskResult = new TaskResultImpl(taskContext.getTaskId(),
-                                                result,
-                                                null,
-                                                stopwatch.elapsed(TimeUnit.MILLISECONDS));
+                        result,
+                        null,
+                        stopwatch.elapsed(TimeUnit.MILLISECONDS));
             } catch (Throwable e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)